server/test/integration/history.test.js

```diff
@@ -0,0 +1,56 @@
+import { createConnectedClient } from '../../src/db/dbClient.js';
+import sinon from 'sinon';
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../../server.js';
+import sinonChai from "sinon-chai";
+import { createTestUser, deleteTestUser }  from '../utils/userManagement.js';
+
+const { expect, assert } = chai;
+chai.use(sinonChai);
+chai.use(chaiHttp);
+
+describe('/api/history integration tests', () => {
+    const sandbox = sinon.createSandbox();
+    let testUserObj;
+    let pgClient;
+  
+    before(async () => {
+      pgClient = await createConnectedClient();
+    })
+  
+    beforeEach(async () => {
+      testUserObj = await createTestUser(pgClient, app);
+    });
+  
+    afterEach(async () => {
+      sandbox.restore();
+      await deleteTestUser(pgClient, testUserObj.userId);
+    });
+  
+    after(() => {
+      pgClient.end();
+    });
+
+    it('should not show history of trades if cookie is missing', async () => {
+        const res = await chai.request(app)
+          .get('/api/history')
+          .set('Content-Type', 'application/json')
+
+        expect(res.status).to.equal(401);
+
+      });
+    
+  
+    it('should show history of trades', async () => {
+        const res = await chai.request(app)
+            .get('/api/history')
+            .set('Content-Type', 'application/json')
+            .set('Cookie', testUserObj.setCookie)
+  
+        expect(res.status).to.equal(200);
+ 
```

Comment:
This test is not enough. You need to check the response body for the actual trades done by this test user.